neurom/check/tests/test_neuron_checks.py

```diff
@@ -222,15 +221,13 @@ def test_nonzero_segment_lengths_bad_data():
              'Single_axon.swc',
              ]
 
-    bad_ids = [[(2, 0), (23, 0), (44, 0), (65, 0)],
-               [(2, 0)],
-               [(2, 0)],
-               [(2, 0)],
-               [(2, 0)]]
+    morphio_offset = 0
+    bad_ids = [[2, 23, 44, 65], [2], [2], [2], [2]]
 
     for i, nrn in enumerate(_pick(files)):
         ids = nrn_chk.has_all_nonzero_segment_lengths(nrn)
-        nt.assert_equal(ids.info, bad_ids[i])
+        nt.assert_equal(ids.info,
+                        [(id + morphio_offset, 0) for id in bad_ids[i]])
```

Comment:
what does this 2nd part represent? does it need `morphio_offset` as well?

Reply:
It represents the point id (not the section id) so it does not require an offset.